doc/pages/expansions.asciidoc

```diff
@@ -353,6 +365,15 @@ The following expansions are supported (with required context _in italics_):
     buffer is modified, including undoing and redoing previous modifications
     (see also `%val{history_id}`)
 
+*%val{uncommitted_modifications}*::
```

Comment:
I wonder if this should be exposed directly as part of `history` as a special last element ? Do you see cases where we would want to get history, but not uncommitted_modifications ?